src/main/scala/day1/StringReplacer.scala

```diff
@@ -0,0 +1,20 @@
+package day1
+
+object StringReplacer extends App {
+
+  def charChecker(inputChar: Char, replaceChar: Char, insertChar: Char): Char = {
```

Comment:
this could just be a if else statement without the need to store the value in memory